bin/common_routines.sh

```diff
@@ -0,0 +1,14 @@
+require_env_var() {
+  if [[ -z ${!1} ]]
+  then
+    echo "$1 env var is needed." && exit 1
+  fi
+}
+
+get_host_from_hostport() {
+  echo $(echo $1 | awk -F':' '{ print $1 }')
+}
+
+get_port_from_hostport() {
+  echo $(echo $1 | awk -F':' '{ print $2 }')
+}
```

Comment:
May be this file can be part of https://github.com/ebi-gene-expression-group/solr-jsonl-loader.git repo as well.. as It seems to be present in all repo which uses `solr-jsonl-chunk-loader.sh`

Reply:
@anilthanki Yes, I was thinking the same thing, but TBH it is nothing to do with solr chunk loading and for example `require_env_var ` function is used by many many shell scripts in different repositories.
Therefore, I would not put into that repo.